app/components/answer-step.jsx

```diff
@@ -0,0 +1,117 @@
+// https://github.com/EnCiv/civil-pursuit/issues/102
+import React, { useState, useEffect } from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+import StepIntro from './step-intro'
+import WhyInput from './why-input'
+import ObjectId from 'bson-objectid'
+
+const AnswerStep = props => {
+  const {
+    className = '',
+    intro = '', // text for the introduction
+    whyQuestion1 = '', // the text for the left of the first why input
+    whyQuestion2 = '', // the text for the left of the second why input
+    question = {}, // a pointObj
+    shared = {
+      startingPoint: {},
+      whyMosts: [],
+    },
+    onDone = () => {},
+    ...otherProps
+  } = props
+
+  const classes = useStylesFromThemeFunction(props)
+  const { startingPoint: initialStartingPoint, whyMosts } = shared
+  const [startingPoint, setStartingPoint] = useState(initialStartingPoint)
+  const whyPoints =
+    Object.keys(initialStartingPoint).length === 0
+      ? whyMosts
+      : whyMosts.filter(point => point.parentId === initialStartingPoint._id)
+  const [whyMostInputs, setWhyMostInputs] = useState(whyPoints)
+
+  useEffect(() => {
+    if (areAnswersComplete(whyMostInputs)) {
+      onDone({ valid: true, value: { startingPoint: startingPoint, whyMosts: whyMostInputs } })
+    } else {
+      onDone({ valid: false, value: { startingPoint: startingPoint, whyMosts: whyMostInputs } })
+    }
+  }, [whyMostInputs, startingPoint])
+
+  const updateWhyResponse = ({ valid, value }) => {
+    let newStartingPoint = startingPoint
+    if (Object.keys(startingPoint).length === 0) {
+      newStartingPoint = {
+        subject: 'Starting Point',
+        description: 'Starting Point Description',
+        _id: ObjectId().toString(),
+      }
+      setStartingPoint(newStartingPoint)
+    }
+
+    const updatedAnswers = whyMostInputs.map(answer => {
+      if (answer._id === value.parentId) {
+        answer.answerSubject = value.subject
```

Comment:
the answer has to follow the pointObj format, meaning subject, description, ... not asnwerSubject.